src/mux.c

```diff
@@ -175,36 +175,37 @@ static bool stick_deviated(mux_channel_t channel)
 
 static bool do_roll_pitch_yaw_muxing(mux_channel_t channel)
 {
+  bool rc_ovrd;
   //Check if the override switch exists and is triggered, or if the sticks have deviated enough to trigger an override
   if ((rc_switch_mapped(RC_SWITCH_ATT_OVERRIDE) && rc_switch(RC_SWITCH_ATT_OVERRIDE)) || stick_deviated(channel))
   {
-    rc_override = true;
```

Comment:
Just for clarity, since the original issue was that function was writing to the static rc_override variable when it should have been writing to the local rc_override variable that it returns, as is done in the throttle muxing function. I just found it confusing having the local and static variables with the same name, so I decided to change it.